src/blocktree.rs

```diff
@@ -399,6 +399,18 @@ impl Blocktree {
         self.meta_cf.get(&MetaCf::key(slot_height))
     }
 
+    pub fn clear_slot(&self, slot_height: u64) -> Result<()> {
+        let meta_key = MetaCf::key(slot_height);
+        if let Some(mut meta) = self.meta_cf.get(&meta_key)? {
+            if meta.consumed > 0 {
```

Comment:
It's possible that there are blobs in this slot, but blob 0 isn't in the slot. In which case, meta.consumed can equal zero, while received > 0, do we care about this case?